Assets/Scripts/GameManager.cs

```diff
@@ -10,6 +10,7 @@ public class GameManager : MonoBehaviour {
     public GameObject bulletPowerupPrefab;
     public GameObject shipControlPowerupPrefab;
     public GameObject doubleShotPowerupPrefab;
+	public GameObject addLifePowerupPrefab;
```

Comment:
ok i fixed it i think... I think stupid Github is doing this crap because I'm not putting in dumb spaces...